aiohttp/web_protocol.py

```diff
@@ -523,24 +524,30 @@ def handle_error(self,
         information. It always closes current connection."""
         self.log_exception("Error handling request", exc_info=exc)
 
-        if status == 500:
-            msg = "<h1>500 Internal Server Error</h1>"
+        ct = 'text/plain'
+        if status == HTTPStatus.INTERNAL_SERVER_ERROR:
+            title = '500 Internal Server Error'
```

Comment:
```suggestion
            title = ' '.join((HTTPStatus.INTERNAL_SERVER_ERROR.value, HTTPStatus.INTERNAL_SERVER_ERROR.description))
```

Reply:
```python
In [1]: import http                                                                                                                                                                            

In [3]: http.HTTPStatus.INTERNAL_SERVER_ERROR                                                                                                                                                  
Out[3]: <HTTPStatus.INTERNAL_SERVER_ERROR: 500>

In [5]: http.HTTPStatus.INTERNAL_SERVER_ERROR.phrase                                                                                                                                           
Out[5]: 'Internal Server Error'

In [6]: http.HTTPStatus.INTERNAL_SERVER_ERROR.value                                                                                                                                            
Out[6]: 500

In [8]: http.HTTPStatus.INTERNAL_SERVER_ERROR.description                                                                                                                                      
Out[8]: 'Server got itself in trouble'
```

Reply:
@webknjaz honestly the original (a plain string) looks more readable for me.

Reply:
But I can live with any variant, too negligeable reason for contestion.

Reply:
Agreed. I'd probably go for
```suggestion
            title = str(HTTPStatus.INTERNAL_SERVER_ERROR.value) + ' ' + HTTPStatus.INTERNAL_SERVER_ERROR.phrase
```

Reply:
would it be better readable?

Reply:
I'm a bit surprised that `http.HTTPStatus` doesn't have a method to construct a status string. IMHO it should be smth like `title = http.HTTPStatus.INTERNAL_SERVER_ERROR.to_status_string()`

Reply:
HTTP reason is gone in htttp/2 BTW.
I suspect that very many Web frameworks have own copies of these constants.
+0 for adding `to_status_string()` to CPython, at least I don't want to make a Pull Request (but you can count me on the PR review).